src/DataViews/PresetsDataView.cpp

```diff
@@ -255,8 +255,8 @@ void PresetsDataView::OnDataChanged() {
     indices_[i] = i;
     std::vector<ModuleView> modules;
     for (const auto& module_path : presets_[i].GetModulePaths()) {
-      modules.emplace_back(module_path.filename().string(),
-                           presets_[i].GetNumberOfFunctionsForModule(module_path));
+      size_t num_functions = presets_[i].GetNumberOfFunctionsForModule(module_path);
+      modules.emplace_back(module_path.filename().string(), num_functions);
```

Comment:
There are some implicit conversions going on here as well. Maybe this can be improved to be either `size_t` or `uint32_t` (not both). 

Either change `GetNumberOfFunctionsForModule` to return `uint32_t` instead of `size_t` (it internally casts `int` to the return type). Or change `ModuleView.function_count` to `size_t`

Reply:
I just scrolled down and saw that you changed `function_count` to `uint64_t`.. maybe still make it uniform to either `size_t` or `uint64_t`

Reply:
Changed `function_count` to `size_t`, thanks